content/md/franklin-insights/franklin-insights-subscriptions.md

```diff
@@ -0,0 +1,38 @@
+---
+id: franklin-insights-subscriptions
+themes: franklin-insights
+title: Franklin Insights **subscriptions**
+ee-only: true
+related: franklin-insights-getting-started, franklin-insights-mappings
+---
+
+# Products subscriptions overview
+
+Once you [activate the Franklin connection](/franklin-insights-getting-started.html#franklin-activation) and perform the [mapping of Franklin identifiers](/franklin-insights-mappings.html#identifiers-mapping), you have everything you need in order to **subscribe products to Franklin for automatic enrichment**.
+
+# Product subscription facts
```

Comment:
Maybe, put this section into the overview as a subsection and rename it to something like "What you need to know about product subscriptions". What do you think?